src/App.tsx

```diff
@@ -90,64 +91,66 @@ export function App() {
+                  <Route path="public/:username" element={<UserPublicPage />} />
+                  <Route path="users" element={<PublicUserList />} />
+                  <Route path="user/:username" element={<UserPublicPage />} />
+                  <Route element={<ProtectedRoutes />}>
+                    <Route index element={<TimestampPhotos />} />
+                    <Route path="things" element={<AlbumThing />} />
+                    <Route path="recent" element={<RecentlyAddedPhotos />} />
+                    <Route path="favorites" element={<FavoritePhotos />} />
+                    <Route path="deleted" element={<DeletedPhotos />} />
+                    <Route path="hidden" element={<HiddenPhotos />} />
+                    <Route path="notimestamp" element={<NoTimestampPhotosView />} />
+                    <Route path="useralbums" element={<AlbumUser />} />
+                    <Route path="places" element={<AlbumPlace />} />
+                    <Route path="people" element={<AlbumPeople />} />
+                    <Route path="events" element={<AlbumAuto />} />
+                    <Route path="statistics" element={<Statistics />} />
+                    <Route path="settings" element={<Settings />} />
+                    <Route path="profile" element={<Profile />} />
+                    <Route path="library" element={<Library />} />
+                    <Route path="faces" element={<FaceDashboard />} />
+                    <Route path="search" element={<SearchView />} />
+                    <Route path="person/:albumID" element={<AlbumPersonGallery />} />
+                    <Route path="place/:albumID" element={<AlbumPlaceGallery />} />
+                    <Route path="thing/:albumID" element={<AlbumThingGallery />} />
+                    <Route path="event/:albumID" element={<AlbumAutoGalleryView />} />
+                    <Route path="explorer" element={<Explorer />} />
+                    <Route path="albumviewer" element={<AlbumViewer />} />
+                    <Route path="useralbum/:albumID" element={<AlbumUserGallery />} />
+                    <Route path="shared/tome/:which" element={<SharedToMe />} />
+                    <Route path="shared/fromme/:which" element={<SharedFromMe />} />
+                    <Route path="admin" element={<AdminPage />} />
+                    <Route path="map" element={<PhotoMap />} />
+                    <Route path="placetree" element={<LocationTree />} />
+                    <Route path="wordclouds" element={<WordClouds />} />
+                    <Route path="timeline" element={<Timeline />} />
+                    <Route path="socialgraph" element={<Graph />} />
+                    <Route path="facescatter" element={<FaceScatter />} />
+                    <Route path="countstats" element={<CountStats />} />
+                  </Route>
+                </Routes>
+              </AppShell>
+            </NotificationsProvider>
+          </ModalsProvider>
```

Comment:
is this just a reformatting or did something change here?

Reply:
I added "ModalsProvider." This is a nice feature that can centralize modals handling. For now, it is needed to properly site the "confirm you want to delete a user" modal. Other than that, these lines are just indentation changes.

Reply:
Looks good to me! :+1: